### PR TITLE
Feature/Blueprints Upgrade

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

Feature/Blueprints Upgrade

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by elamaran11** [comment]
/do-e2e-test single-new-eks-fargate-opensource-observability deploy

**Comment by iamprakkie** [comment]
LGTM

**Review by iamprakkie — DISMISSED** [review]
LGTM

**Comment by elamaran11** [comment]
/do-e2e-test single-new-eks-fargate-opensource-observability destroy

File: lib/single-new-eks-opensource-observability-pattern/index.ts [hunk 1]
```diff
@@ -173,12 +171,17 @@ export default class SingleNewEksOpenSourceobservabilityPattern {
             addOns.push(new blueprints.addons.IstioControlPlaneAddOn({
                 version: "1.18.2"
             }));
-            addOns.push(new IstioIngressGatewayHelmAddon);
-            addOns.push(new IstioCniHelmAddon);
+            addOns.push(new blueprints.addons.IstioIngressGatewayAddon({
+                version: "1.18.2"
```
Comment (shapirov103) [inline lib/single-new-eks-opensource-observability-pattern/index.ts:175]: is specifying the exact version important here? I see other addons don't specify it, is it in this case different from the default?

**Review by shapirov103 — COMMENTED** [review]
@elamaran11 looks great, a couple of minor comments/questions

File: lib/single-new-eks-opensource-observability-pattern/graviton-index.ts [hunk 2]
```diff
@@ -170,12 +168,17 @@ export default class SingleNewEksGravitonOpenSourceObservabilityPattern {
...
+            }));
+            
+            addOns.push(new blueprints.addons.IstioCniAddon({
+                version: "1.18.2"
+            }));
         }
 
         const mngProps: blueprints.MngClusterProviderProps = {
-            version: eks.KubernetesVersion.of("1.27"),
+            version: eks.KubernetesVersion.of("1.29"),
```
[the remark this answers is not in the document]
Comment (elamaran11) [reply lib/single-new-eks-opensource-observability-pattern/graviton-index.ts]: Fixed.

File: lib/single-new-eks-opensource-observability-pattern/index.ts (hunk 1 above)
Comment (elamaran11) [reply lib/single-new-eks-opensource-observability-pattern/index.ts:175]: Yes this particular demonstration we are doing works only with this version or the example setup changes. So we are sticking to this version for now.

**Comment by elamaran11** [comment]
> @elamaran11 looks great, a couple of minor comments/questions

@shapirov103 Fixed, Can you apporve this. Im going to run couple of e2e patterns and will merge.

**Review by iamprakkie — APPROVED** [review]
LGTM

**Pull request merged**